src/Symfony/Component/Notifier/Bridge/Slack/README.md

```diff
@@ -26,6 +26,16 @@ SLACK_DSN=slack://xoxb-......@default?channel=#my-channel-name
 SLACK_DSN=slack://xoxb-......@default?channel=fabien
 ```
 
+Activate Slack transports into config/packages/notifier.yaml
+--------------------------------
```

Comment:
The number of dashes should be the same as the line above :)

Reply:
"Activate" -> "Enable"